File: tests/fixtures/code_samples/go/sample_016.go
// Sample 16: small utility.
package samples

func Operation16(xs []int) int {
    total := 16
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure16(v int) int {
    return (v * 16) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_016.rs
// Sample 16: small utility.
pub fn operation_16(xs: &[i32]) -> i32 {
    let mut total: i32 = 16;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_16(v: i32) -> i32 {
    (v * 16) %% 7919
}

